core/templates/components/main-nav/main-nav.twig

```diff
@@ -0,0 +1,388 @@
+
+  {% if has_search == true %}
```

Comment:
Adding the modifier class when the search is enabled should be as easy as adding the class to `{{ modifier_class }}` instead of having the main nav wrapper in the if statement as an html tag. Less markup in the template and easier to maintain/read imho.

Reply:
Sounds good - will do!